examples/seismic/viscoelastic/viscoelastic_example.py

```diff
@@ -1,14 +1,14 @@
 import numpy as np
 from argparse import ArgumentParser
 
+from devito import configuration
 from devito.logger import info
 from examples.seismic.viscoelastic import ViscoelasticWaveSolver
 from examples.seismic import demo_model, AcquisitionGeometry
 
 
-def viscoelastic_setup(shape=(50, 50), spacing=(15.0, 15.0), tn=500.,
-                       space_order=4, nbpml=10,
-                       constant=True, **kwargs):
+def viscoelastic_setup(shape=(50, 50), spacing=(15.0, 15.0), tn=500., space_order=4,
+                       nbpml=10, constant=True, **kwargs):
```

Comment:
@rhodrin @mloubout note that `constant=True` here, while it's `constant=False` in elastic

Reply:
We talked about that in slack, @rhodrin  did that in purpose.